### DISCUSSION TITLE
Vercel Deployment failed

### DISCUSSION THREAD

**Discussion by PsclDev** [anuraghazra/github-readme-stats#1383] [Q&A] [answered]
Everytime I try to deploy the repo, I've got a error Message:
"...Max Duration has to be between 1 and 10 seconds..."

Does anybody know if I need a paid plan on vercel to run this?

**Comment by PsclDev** [comment, marked as answer]
Okay, I found the solution for this issue.

Inside the vercel.json you have to change 
`"maxDuration": 30` to `"maxDuration": 10`.

After that you can redeploy on vercel and it will work with the free plan

**Reply by Tanishq404E** [reply]
i see 👍

**Reply by rickstaa** [reply]
The reason why this needs to be done is found in #1416.
